src/jekyll/_plugins/custom_tags/audio.rb

```diff
@@ -0,0 +1,59 @@
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
Just to be in-keeping with the ytvideo embed code, and @gauntface said it was his preference, though I guess I can leave you two to fight that out? :runner: